doc/source/conf.py

```diff
@@ -32,6 +32,7 @@
         "version_match": get_version_match(__version__),
     },
     "check_switcher": False,
+    "navigation_with_keys": False,
```

Comment:
With the new Sphinx theme you shouldn't need it. It has already been released. Bump your dependency